playground/my_playground.py

```diff
@@ -0,0 +1,99 @@
+import pandas as pd
+from pprint import pprint
+import sys
+import sqlite3
+
+def read_data(filename):
+    """(pd.read_csv) uses pandas to read a csv file and convert it to a dataframe
+
+    Args:
+        filename (string): a csv filename 
+    """
+    dataframe = pd.read_csv(filename)
+    print('Printing dataframe...')
+    print(dataframe)
+    return dataframe
+    
+def df_iter_rows(df):
+    """(dataframe.iterrows) uses dataframe.iterrows to print first 3 rows in df
+
+    Args:
+        df (dataframe): pandas dataframe
+    """
+    print('(ITER_ROWS) Printing first 3 rows in the dataframe...')
+    for index, row in df.iterrows():
+        if index == 3: break
+        print('ROW:', index)
+        print(row)
+    
+def df_iter_tuples(df):
+    """(dataframe.itertuples) uses dataframe.itertuples to print first 3 row tuples in df
+
+    Args:
+        df (dataframe): pandas dataframe
+    """
+    index = 0
+    print('(ITER_TUPLES) Printing first 3 tuples in the dataframe...')
+    for tuple in df.itertuples():
+        if index == 3: break
+        print('TUPLE:', index)
+        print(tuple)
+        index = index + 1 
+
+def df_iter_columns(df):
+    """converts df into a python list to print first 3 columns in df
+
+    Args:
+        df (dataframe): pandas dataframe
+    """
+    columns = list(df)
+    index = 0
+    print('(COLUMNS) Printing first 3 columns...')
+    for i in columns:
+        if index == 3: break
+        print('COLUMN:', index)
+        print (df[i])
+        index = index + 1
+
+def load_file_into_db(filename, table, conn):
+    """(dataframe.to_sql) load a table into a database connection istance
```

Comment:
Nit: `load` -> `loads`

[Google uses descriptive](https://google.github.io/styleguide/pyguide.html#244-decision), rather than imperative mood for function descriptions.